src/styles/calendar.scss

```diff
@@ -0,0 +1,56 @@
+.calendar {
+  /*styles for calendar */
```

Comment:
Remove all comments. Try it write obvious code that explain all we need without comments)